### PR TITLE
[FEAT] bump compiler version

### PR DESCRIPTION
**Pull request by Hugo-Inmanta**
# Description

Bump compiler version 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Type annotations are present
- [ ] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: src/inmanta/__init__.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
     Contact: code@inmanta.com
 """
 
-COMPILER_VERSION = "2020.7.2"
+COMPILER_VERSION = "2020.8"
```
Comment (sanderr) [inline src/inmanta/__init__.py:19]:
```suggestion
COMPILER_VERSION = "2020.7.3"
```

File: changelogs/unreleased/release-bump-compiler-version.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+description: Bump compiler version to 2020.8
+change-type: patch
+destination-branches: [iso4]
+sections:
+  minor-improvement: "{{description}}"
```
Comment (sanderr) [inline changelogs/unreleased/release-bump-compiler-version.yml:5]: No need for this
```suggestion
```

**Review by sanderr — CHANGES_REQUESTED** [review]

**Comment by Hugo-Inmanta** [comment]
No compiler fixes in iso4, no need for a compiler version bump

File: changelogs/unreleased/release-bump-compiler-version.yml (hunk 2 above)
Comment (arnaudsjs) [inline changelogs/unreleased/release-bump-compiler-version.yml:5]: Isn't it weird to add this message to the changelog?

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**